src/state/runner/sagas/UpdateSimulationRunner.js

```diff
@@ -33,7 +33,9 @@ export function* callUpdateRunner(action, throwOnError = false) {
       runnerId,
       runnerDataForRequest
     );
-    runnerData.lastUpdate = updateData.lastUpdate;
+    const updateInfo =
+      updateData?.updateInfo ?? (updateData?.lastUpdate != null ? { timestamp: updateData.lastUpdate } : undefined);
+    if (updateInfo != null) runnerData.updateInfo = updateInfo;
```

Comment:
Since `updateData` is the API response, [starting with v5](https://github.com/Cosmo-Tech/cosmotech-api/blob/5.0.0-beta6/doc/Models/Runner.md), it shouldn't have any field `lastUpdate`. 

We don't need to support retro-compatibility with prior versions of the API, so we can probably simply write
`runnerData.updateInfo = updateData.updateInfo;`